Makefile

```diff
@@ -6,67 +7,34 @@ GIT_VERSION?=$(shell git describe --always --long --abbrev=8 --dirty)
 GO_LDFLAG_VERSION := -X 'main.Version=$(GIT_VERSION)'
 MIST_COMMIT ?= "catalyst"
 DOCKER_TAG ?= "livepeer/catalyst"
+FROM_LOCAL_PARENT ?= "scratch" # for `make docker-local` and `make box-local`
+DOCKER_TARGET ?= "catalyst"
 BUILD_TARGET ?= "full"
+KILL ?= "false"
 
 $(shell mkdir -p ./bin)
 $(shell mkdir -p ./build)
+$(shell mkdir -p ./data)
+$(shell mkdir -p ./coredumps)
 $(shell mkdir -p $(HOME)/.config/livepeer)
```

Comment:
```
data
build
bin
```
are already in the `.gitignore`.
I think we should add them into the repo and gitignore their contents... no need to create them dynamically

Reply:
The way I usually see that implemented is with a `.gitkeep` file or something in the directory. But we don't want to do that for `bin` or we'll ship `.gitkeep` to everyone!